pkg/middlewares/aws_security_group_defaults.go

```diff
@@ -0,0 +1,52 @@
+package middlewares
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+const defaultAwsSecurityGroupName = "default"
+
+// When scanning a brand new AWS account, some users may see irrelevant results about default AWS security group.
+// We ignore these resources by default when strict mode is disabled.
+type AwsSecurityGroupDefaults struct{}
```

Comment:
We already have handled these cases in `VPCDefaultSecurityGroupSanitizer`

Reply:
Yep, it's already done